index.html

```diff
@@ -237,5 +233,61 @@ <h3 class="large">Skills</h3>
         <img src="images/ic_skills.png" class="small-image" alt="">
       </div>
       </div>
+      <footer class="footer">
+        <div class="background-img1">
+        <img src="images/image_geometry_1@2x.png" class="background-image" alt=""> 
+        </div>
+
+       
+          <h3 class="footer-header">Get started</h3>
+
+          <p class="footer-p">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+      
+
+      <div >
+        <form class="form" action="https://formspree.io/f/mqkngogp" method="POST">
+
+          <div class="form-container">
+            <label class="form-label" for="Username"></label>
+            <input type="text" class="name" id="Username" name="Username" placeholder="Your name" maxlength="30" required>
+           </div>
+
+           <div class="form-container">
+             <label class="form-label" for="email"></label>
+            <input type="email" class="email" id="email" name="email"  placeholder="Your email" required>
+            </div>
+
+            <div class="form-container">
+              <label class="form-label" for="message"></label>
+            <textarea class="text-area" id="message" name="Message" required rows="10" maxlength="500" placeholder="Enter your message here..."></textarea>
+            </div>
+
+        <div class="start-btn-div">
+          <button type="submit" class="start-btn-2">Start collaboration</button>
```

Comment:
- [ ] Kindly add button interactions such as hover, active (when it's pressed), disabled, and enable (default state) as styled  in the Figma template see screenshot below 🤝 .

 
![Screenshot_136](https://user-images.githubusercontent.com/69638013/167097506-715b34bb-4ae0-46f6-9806-43de7cfe5dc5.png)